contracts/dtoken/src/borrow.rs

```diff
@@ -65,10 +65,9 @@ impl Contract {
     pub fn make_borrow_callback(&mut self, token_amount: WBalance) -> PromiseOrValue<WBalance> {
         if !is_promise_success() {
             log!(
-                "failed to borrow {} for account {}",
-                token_amount.0,
-                env::signer_account_id()
-            );
+                r#"EVENT_JSON:{{"standard": "nep297", "version": "1.0.0", "event": "borrow_fail", "data": {{"account_id": "{}", "amount": "{}", "reason": "failed to make borrow for {} on {} token amount"}}}}"#,  
```

Comment:
Why don't we create a simple wrapper smth like:

fn custom_log(event: String, data: String) {
    log!(
        r#"EVENT_JSON:{{"standard": "nep297", "version": "1.0.0", "event": {}, "data": {} }}"#,
        event,
        data
    )
}
At the same time we can manipulate by such variable like version etc....
   )
}